jenkins_jobs/trigger_weekly_host_os_build/script.sh

```diff
@@ -1,16 +1,17 @@
 set -e
 
-VERSIONS_REPOSITORY_URL="https://github.com/${GITHUB_ORGANIZATION_NAME}/versions.git"
-PUSH_URL_PREFIX="ssh://git@github.com/${GITHUB_BOT_USER_NAME}"
+MAIN_REPO_URL_PREFIX="https://github.com/${GITHUB_ORGANIZATION_NAME}"
```

Comment:
This is ambiguous. (main repo) url or main (repo url) ? Isn't a "repo url" a remote?